Library/Homebrew/test/cmd/--prefix_spec.rb

```diff
@@ -21,7 +21,14 @@
   end
 
   it "errors if the given Formula doesn't exist", :integration_test do
-    expect { brew "--prefix", "--installed", "nonexistent" }
+    expect { brew "--prefix", "nonexistent" }
+      .to output(/No available formula/).to_stderr
+      .and not_to_output.to_stdout
+      .and be_a_failure
+  end
+
+  it "errors if the given Formula actually matches a Cask", :integration_test do
```

Comment:
Let's not add another integration test for this.

Reply:
So are you asking to:
1. drop the "google-chrome" test entirely
2. replace the "nonexistent" test with "google-chrome"
3. something else?

Reply:
I think: don't add/change the tests in this PR at all.

Reply:
OK, but I'd argue that:
```
   it "errors if the given Formula doesn't exist", :integration_test do
    expect { brew "--prefix", "--installed", "nonexistent" }
```
really should be:
```
   it "errors if the given Formula doesn't exist", :integration_test do
    expect { brew "--prefix", "nonexistent" }
```
based on the test description.